wsagent/che-core-api-git/src/test/java/org/eclipse/che/git/impl/CommitTest.java

```diff
@@ -108,4 +111,33 @@ public void testAmendCommit(GitConnectionFactory connectionFactory) throws GitEx
         assertEquals(revision.getMessage(), commitRequest.getMessage());
         assertEquals(beforeCount, afterCount);
     }
+
+    @Test(dataProvider = "GitConnectionFactory", dataProviderClass = org.eclipse.che.git.impl.GitConnectionFactoryProvider.class,
+          expectedExceptions = GitException.class)
```

Comment:
The problem is that Native Git and JGit are giving different messages in this testcase